site/themes/contour/layouts/partials/team.html

```diff
@@ -1,7 +1,7 @@
 <div class="team">
     <div class="wrapper">
         <h2>The Contour Team</h2>
-        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="#">GitHub issues page for Contour</a>.</p>
+        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="https://github.com/vmware-tanzu/community-edition/issues/new/choose">GitHub issues page for Contour</a>.</p>
```

Comment:
```suggestion
        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="https://github.com/projectcontour/contour/issues/new/choose">GitHub issues page for Contour</a>.</p>
```

Reply:
Oops, looks like the fix for this got reverted in the latest commit, can you reapply the fix?